flipperdevices/flipperzero-firmware
# [FL-1214] NFC application

# What's new

- Introduce api-hal-nfc
- Move nfc from services to applications
- Rework nfc worker
- Add system on start hooks

# Verification 

- Build for f5 target
- Verify nfc detection
- Verify nfc_detect cli command

# Checklist (do not modify)
